lib/chirp_cockroach_web/channels/user_socket.ex

```diff
@@ -0,0 +1,54 @@
+defmodule ChirpCockroachWeb.UserSocket do
```

Comment:
Is this socket used by Absinthe? I see it's set up in `assets/js/user_socket.js`, but it doesn't seem to be used in any specific way. Or is it just a scaffolding from testing a basic websocket setup?

Asking out of curiosity, as it seems to me there's dedicated Apollo-Absinthe code for the GraphQL interface independent of this.

Reply:
`use Absinthe.Phoenix.Socket` defines a channel inside Socket - so this socket can have additional extra channels that are independent of Absinthe. This implementation of GraphQL websocket endpoint is not "universal" implementation - it relies on "@absinthe/socket" and  "@absinthe/socket-apollo-link" JS libraries (see `apollo/index.js` - also it's good to check it in next Authorization PR)

`assets/js/user_socket.js` is autogenerated code - I left it for future reference, especially ChirpCockroach is kind of "general ideas" project, so maybe some part

Reply:
All clear, thanks 👍